packages/@aws-cdk/core/lib/asset-staging.ts

```diff
@@ -175,17 +175,22 @@ export class AssetStaging extends Construct {
         hostPath: bundleDir,
         containerPath: AssetStaging.BUNDLING_OUTPUT_DIR,
       },
-      ...options.volumes ?? [],
+      ...options.docker.volumes ?? [],
     ];
 
     try {
       process.stderr.write(`Bundling asset ${this.construct.path}...\n`);
-      options.image._run({
-        command: options.command,
+
+      if (options.local?.tryBundle(bundleDir)) {
```

Comment:
I think we can pass in `options` as a whole for `tryBundle` to be able to reuse some config from there